content/foundations/physical_properties/Lists_of_duplicates.rst

```diff
@@ -0,0 +1,10 @@
+Lists of duplicates:
+	pages:
+		-magnetics_susceptibility_duplicate.rst
+		-seismic_velocity_duplicate.rst
+		-induced_polarization_physical_properties_duplicate.rst
+		-susceptibility_duplicate.html 
+
+		Pages have been duplicates due to a problem with the navbar on the website. The students were sent to the original page and so were leaving the "foundations" chapter. This solution is meant to be temporary until a more suitable one have been found.	
+
+	all duplicates images have been stored in ./images/images_duplicates/. This choice has been made to ensure that any displacement of the physical properties section would not require (as it happened before) to relocate all images links.
```

Comment:
could you create an issue on github instead? (It will be easier to keep track of and see that we need to take care of it later on)